lte/protos/session_manager.proto

```diff
@@ -793,29 +843,33 @@ enum SMSessionFSMState {
 //PDU session Establishment from AMF to SMF
 //SET MESSSAGE
 message M5GSMSessionContext {
-  bytes pdu_session_id = 1;   //One byte
-  RequestType rquest_type = 2;        //Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
-  RedirectServer pdu_address = 3; //IP address. Imported from session_manager.proto file
+  uint32 pdu_session_id = 1;
+  RequestType rquest_type = 2;        // Define ENUM 24-501 - 9.11.3.47 . 1 byte. i.e. initial requerst, existing pdusession etc...
```

Comment:
typo in the comment `requerst`. Also there is no much saving here in field name, why not use `request_type` rather than omitting an e?

Reply:
resolved